deCODE/deCODE.py

```diff
@@ -0,0 +1,132 @@
+#!/usr/bin/env python3
+
+# Import library for analysis-runner
+import click
+import hail as hl
+from cpg_utils.hail_batch import output_path
+
+# Import library for deCODE job
+import pandas as pd
+from pandas import Series,DataFrame
+import numpy as np
+import math
+import collections
+
+@click.command()
+@click.option('--dataset', help="data to query")
+@click.option('--output', help="output name")
+@click.option('--rerun', help='Whether to overwrite cached files', default=False)
```

Comment:
click has a parameter for flags:

```suggestion
@click.option('--rerun', is_flag=True, help='Whether to overwrite cached files', default=False)
```